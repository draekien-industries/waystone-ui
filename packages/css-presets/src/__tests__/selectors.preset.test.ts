import { cssSelectors } from '../selectors.presets';

describe('CSS Selector presets', () => {
  it('cssSelectors should match snapshot', () => {
    expect(cssSelectors).toMatchSnapshot();
  });
});
