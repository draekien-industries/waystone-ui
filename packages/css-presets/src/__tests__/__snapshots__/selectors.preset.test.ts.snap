// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSS Selector presets cssSelectors should match snapshot 1`] = `
{
  "active": ":enabled:active",
  "disabled": ":disabled",
  "focusVisible": ":enabled:not(active):focus-visible",
  "hover": ":enabled:hover",
}
`;
